packages/internal-input-option/src/InputOption.style.ts

```diff
@@ -0,0 +1,125 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import {
+  fontFamilies,
+  spacing,
+  transitionDuration,
+  typeScales,
+} from '@leafygreen-ui/tokens';
+
+export const inputOptionStyles = css`
+  position: relative;
+  list-style: none;
+  cursor: pointer;
+  outline: none;
+  border: unset;
+  margin: 0;
+  text-align: left;
+  text-decoration: none;
+
+  font-size: ${typeScales.body1.fontSize}px;
+  line-height: ${typeScales.body1.lineHeight}px;
+  font-family: ${fontFamilies.default};
+  padding: ${spacing[2]}px ${spacing[2] + spacing[1]}px;
+
+  transition: background-color ${transitionDuration.default}ms ease-in-out;
+
+  &:focus,
+  &:focus-visible {
+    outline: none;
+    border: unset;
+  }
+`;
+
+export const inputOptionThemeStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    color: ${palette.black};
+    background-color: ${palette.white};
+
+    &:hover {
+      outline: none;
+      background-color: ${palette.gray.light2};
+    }
+  `,
+  [Theme.Dark]: css`
+    color: ${palette.white};
+    background-color: ${palette.gray.dark3};
+
+    &:hover {
+      outline: none;
+      background-color: ${palette.gray.dark4};
+    }
+  `,
+};
+
+/** in px */
+const wedgeWidth = spacing[1];
+/** in px */
+const wedgePaddingY = spacing[2];
+
+export const inputOptionWedge = css`
+  // Left wedge
+  &:before {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 50%;
+    width: ${wedgeWidth}px;
+    height: calc(100% - ${2 * wedgePaddingY}px);
+    background-color: rgba(255, 255, 255, 0);
+    border-radius: 0 6px 6px 0;
+    transform: scale3d(0, 0.3, 0) translateY(-50%);
+    transform-origin: 0%; // 0% since we use translateY
+    transition: 200ms ease-in-out;
```

Comment:
@audroue we use 150ms or 300ms in most places -- can we use one of those values here?